logpyle/__init__.py

```diff
@@ -1391,7 +1391,7 @@ def __call__(self) -> Optional[float]:
 
 
 class CPUTime(LogQuantity):
```

Comment:
We could maybe think about renaming this to `WallTime` and `t_wall`.

Reply:
I'd be supportive of that.